src/index.html

```diff
@@ -1,103 +1,212 @@
       </div>
     </section>
-    <main class="hidden">
-      <form class='search'> 
-        <div class='search-container'>
-          <label class='search-options-label' for='search-recipe-select'>Select</label>
-          <select class='search-options'name='search-recipe-options'id='search-recipe-select'>
-            <option value='recipe'>Recipe Name</option>
-            <option value='ingredient'>Ingredient</option>
-          </select> 
-          <input class='search-bar' type='text' placeholder='Search Recipes Here'>
-          <button class='button' id='searchRecipes'>Search Recipes</button> 
-        </div>  
-      </form>
-      <form class='recipe-tags'>
-        <h2 class='recipe-tags-title'>Filter Recipes</h2>
-        <div class='recipe-tag'>
-          <input type='radio' id='starter' name='recipe-tag' value='starter'>
-          <label class='recipe-tag-label' for='starter'>starter</label>
-        </div>
-        <div class='recipe-tag'>
-          <input type='radio' id='appetizer' name='recipe-tag' value='appetizer'>
-          <label class='recipe-tag-label' for='appetizer'>appetizer</label>
-        </div>
-        <div class='recipe-tag'>
-          <input type='radio' id='sideDish' name='recipe-tag' value='side-dish'>
-          <label class='recipe-tag-label' for='side-dish'>side dish</label>
-        </div>
-        <div class='recipe-tag'>
-          <input type='radio' id='snack' name='recipe-tag' value='snack'>
-          <label class='recipe-tag-label' for='snack'>snack</label>
-        </div>
-        <div class='recipe-tag'>
-          <input type='radio' id='breakfast' name='recipe-tag' value='breakfast'>
-          <label class='recipe-tag-label' for='breakfast'>breakfast</label>
-        </div>
-        <div class='recipe-tag'>
-          <input type='radio' id='brunch' name='recipe-tag' value='brunch'>
-          <label class='recipe-tag-label' for='brunch'>brunch</label>
-        </div>
-        <div class='recipe-tag'>
-          <input type='radio' id='lunch' name='recipe-tag' value='lunch'>
-          <label class='recipe-tag-label' for='lunch'>lunch</label>
-        </div>
-        <div class='recipe-tag'>
-          <input type='radio' id='dinner' name='recipe-tag' value='dinner'>
-          <label class='recipe-tag-label' for='dinner'>dinner</label>
-        </div>
-        <div class='recipe-tag'>
-          <input type='radio' id='salad' name='recipe-tag' value='salad'>
-          <label class='recipe-tag-label' for='salad'>salad</label>
-        </div>
-        <div class='recipe-tag'>
-          <input type='radio' id='sauce' name='recipe-tag' value='sauce'>
-          <label class='recipe-tag-label' for='sauce'>sauce</label>
+    <main>
+      <section class="hidden">
+        <form class="search">
+          <div class="search-container">
+            <label class="search-options-label" for="search-recipe-select"
+              >Select</label
+            >
```

Comment:
Not sure if this is just gitHub spacing, but this closing `>` tag should be on the same line as label

Reply:
I take this back. I think I like it!